### PR TITLE
Tribes bounty idea email template

### LINKED ISSUE #6205: Tribes Emails - Funders #1: Bounty Idea Notification
@iamonuwa @PixelantDesign 

An email to notify funders that they have a new idea from "Suggest a Bounty"

Funders #1 - Bounty idea notification

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description
Add tribes bounty idea email template

##### Refers/Fixes
#6205 

PR includes
1. Sales funnel
2. Bounty idea notification
3. Hook & value email

##### Testing

Screenshot from 2020-03-14 12-48-59
Screenshot from 2020-03-14 12-49-31

### CODE REVIEW

**Comment by iamonuwa** [comment]
@PixelantDesign @willsputra please take a look

File: app/retail/templates/emails/tribes/weekly_email.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+          <span><a href="{% url "profile" interested.profile.handle %}"> {{ interested.profile.handle }}</a> {% trans "is willing to build for:" %}</span>
+          <div style="display: flex; justify-content: center; margin-top: 3em;">
+            <div style="background-color: #E7F0FA; color: #3F00FF; padding: 1em; margin-right: 1em">
+              {{ bounty.value_true }} {{ bounty.token_name }}
+            </div>
+            <div style="background-color: #D6FBEB; color: #00A55E; padding: 1em;">
+              {{ bounty.value_in_usdt_now }} USD
+            </div>
+          </div>
+      </div> -->
```
Comment (thelostone-mc) [inline app/retail/templates/emails/tribes/weekly_email.html:136]: why this commented out ? could we remove it ?

File: app/retail/templates/emails/tribes/weekly_email.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    <hr />
+    <div>
+      <h3>{% trans "Bounty ideas for your launchpad" %}</h3>
+      <div>
+        <p>{% trans "New bounty ideas received:" %}</p>
+        <p>12</p>
+      </div>
+      <div>
+        <h3>{% trans "Featured idea" %}</h3>
+        <!-- <div id="bounty_information">
```
Comment (danlipert) [inline app/retail/templates/emails/tribes/weekly_email.html:117]: Why is this section commented out? Do we still need it?

File: app/retail/emails.py [hunk 3]
```diff
@@ -1381,6 +1444,30 @@ def gdpr_reconsent(request):
...
+    return HttpResponse(response_html)
+
+@staff_member_required
+def tribes_hook_value(request):
+    response_html = render_tribes_hook_value(settings.CONTACT_EMAIL)
+    return HttpResponse(response_html)
+
+@staff_member_required
+def tribes_weekly_email(request):
+    from dashboard.models import Bounty, Interest
```
Comment (danlipert) [inline app/retail/emails.py:1459]: Please move all of these imports to the top of the file

**Comment by iamonuwa** [comment]
Going to make changes here today

**Comment by thelostone-mc** [comment]
Closing this out for now but do reopen it once this conflicts + comments are addressed

**Pull request closed without merging**